src/rollback_to_stable/rts_history.c

```diff
@@ -59,9 +62,10 @@ __wt_rts_history_delete_hs(WT_SESSION_IMPL *session, WT_ITEM *key, wt_timestamp_
         if (!dryrun) {
             __wt_verbose_level_multi(session, WT_VERB_RECOVERY_RTS(session), WT_VERBOSE_DEBUG_3,
               WT_RTS_VERB_TAG_HS_UPDATE_REMOVE
-              "deleting history store update with stop_timestamp=%s greater than "
+              "deleting history store update from btree_id=%" PRIu32
```

Comment:
How about something like this `deleting history store update for/of btree_id ... with stop_timestamp`?

Reply:
Yeah that reads a little nicer. Done in 281dc51.